example_data/sample_bundle.json

```diff
@@ -0,0 +1,738 @@
+                "code": "laboratory",
+                "display": "Laboratory"
+              }
+            ]
+          }
+        ],
+        "effectiveDateTime": "2020-10-22",
+        "note": [
+          {
+            "authorString": "genomic interpretation",
+            "text": "Next generation sequence analysis indicates that Maria321 BrighGosh321 is heterozygous for the SMC1A c.1263del p.(Lys422Serfs*28) pathogenic variant that has been confirmed by Sanger sequence analysis (see technical information below)."
+          }
+        ],
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "LA6577-6",
+              "display": "Negative"
+            }
+          ],
+          "text": "report finding"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Observation?_id=20RG-205G0047-overall-interpretation"
+      }
+    },
+    {
+      "fullUrl": "ServiceRequest/1775336d-71ea-cd05-49a3-e80e966e1277",
+      "resource": {
+        "resourceType": "ServiceRequest",
+        "id": "1775336d-71ea-cd05-49a3-e80e966e1277",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/StructureDefinition/servicerequest"
+          ]
+        },
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\">Lab Procedure from RedCap FHIR app</div>"
+        },
+        "status": "active",
+        "intent": "order",
+        "instantiatesCanonical": [
+          "PlanDefinition/20RG-205G0047-plan-definition"
+        ],
+        "performerType": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "310049001",
+              "display": "Clinical genetics service"
+            }
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/gosh-redcap-40388914"
+        },
+        "specimen": [
+          {
+            "type": "Specimen",
+            "reference": "Specimen/specimen-20RG-205G0047"
+          }
+        ],
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/practitioner1-20RG-205G0047"
+          },
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/practitioner2-20RG-205G0047"
+          }
+        ],
+        "category": [
+          {
+            "coding": [
+              {
+                "system": "http://snomed.info/sct",
+                "code": "108252007",
+                "display": "Laboratory procedure"
+              }
+            ]
+          }
+        ],
+        "reasonCode": [
+          {
+            "coding": [
+              {
+                "system": "http://snomed.info/sct",
+                "code": "230387008",
+                "display": "Reason for recommending early-onset benign childhood epilepsy"
```

Comment:
This coding system should be using PanelApp panels as we have done in DiagnosticReport